src/common/components/forms/Textarea.tsx

```diff
@@ -22,17 +22,18 @@ export default function CustomTextarea({
   const variants = {
     default: (
       <VStack>
-        <Text type="p" alignSelf="flex-start">
+        <Text type="p" alignSelf="flex-start" color="#F8F5F1">
```

Comment:
There are changes on this file on the latest `develop`, better do a merge first. xD

Reply:
Merged